components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -514,6 +515,103 @@ bool PublisherInfoDatabase::InsertOrUpdateActivityInfos(
   return transaction.Commit();
 }
 
+uint64_t PublisherInfoDatabase::GetLastReconcileDate() {
+  DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
+
+  bool initialized = Init();
+  DCHECK(initialized);
+  if (!initialized) {
+    return 0;
+  }
+
+  std::string query =
+      "SELECT DISTINCT MAX(date) FROM contribution_info WHERE category = ?";
+  sql::Statement info_sql(db_.GetUniqueStatement(query.c_str()));
+  int column = 0;
+  info_sql.BindInt(column++, ledger::REWARDS_CATEGORY::AUTO_CONTRIBUTE);
+
+  if (info_sql.Step()) {
+    return info_sql.ColumnInt64(0);
+  }
+  return 0;
+}
+
+uint64_t PublisherInfoDatabase::GetAutoContributeCount(
+    uint64_t reconcile_stamp) {
+  DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
+
+  bool initialized = Init();
+  DCHECK(initialized);
+  if (!initialized || reconcile_stamp == 0) {
+    return 0;
+  }
+
+  std::string query =
+    "SELECT DISTINCT COUNT(publisher_id) FROM activity_info AS ai "
```

Comment:
nit: 4 spaces for continuation of a line

Reply:
Fixed